ui/src/components/helpers/WelcomeTutorialFunctions.ts

```diff
@@ -0,0 +1,33 @@
+const tutorialKey = "tutorial_timestamp";
+
+function setTimeStamp() {
+  const dateItem = {
+    value: "value",
+    expiry: new Date().getTime(),
+  };
+  localStorage.setItem(tutorialKey, JSON.stringify(dateItem));
+}
+
+function checkIfValidTimestamp(totalTimeout) {
+  const oldTimestampStr = localStorage.getItem(tutorialKey);
+  if (!oldTimestampStr) {
+    setTimeStamp();
+    return true;
+  }
+  const oldTimestamp = JSON.parse(oldTimestampStr);
+  const curTimestamp = new Date();
+  if (curTimestamp.getTime() - oldTimestamp.expiry > totalTimeout) {
+    localStorage.removeItem(tutorialKey);
+    setTimeStamp();
+    return true;
+  }
+  return false;
+}
+
+const welcomeTutorialFunctions = {
+  tutorialKey,
+  setTimeStamp,
+  checkIfValidTimestamp,
+};
```

Comment:
I don't think we need to put this into an object, I'd rather just export all these individually.